src/test/elements/sageone/contact-payments.js

```
@@ -0,0 +1,29 @@
+'use strict';
+
+const suite = require('core/suite');
+const chakram = require('chakram');
+const tools = require('core/tools');
+const expect = chakram.expect;
+
+const payload = tools.requirePayload(`${__dirname}/assets/contactPayments-create.json`);
+const updatePayload = tools.requirePayload(`${__dirname}/assets/contactPayments-update.json`);
+
+const options = {
+  churros: {
+    updatePayload
+  }
+};
+
+suite.forElement('finance', 'contact-payments', { payload }, (test) => {
+  test.withOptions(options).should.supportCruds(chakram.put);
+  test.should.supportPagination();
+  test
+    .withName(`should support searching ${test.api} by search`)
+    .withOptions({ qs: { where: `id='aed404dc4fc611e695ab0ed51759bcc7'` } })
```

Please use a date value with `<` or `>` sign. This hardcoded value might result in failure of test case after some time when this particular Id will not be present in the system.